flight/targets/coptercontrol/fw/pios_config.h

```diff
@@ -119,11 +119,8 @@
 /* PIOS Initcall infrastructure */
 #define PIOS_INCLUDE_INITCALL
 
+#define SMALLF1
 #define COPTERCONTROL
-// Conditional related to making CC run navigation
-#if defined(NAVGIATION)
-#define PIOS_GPS_PROVIDES_AIRSPEED
```

Comment:
this variable is still used in GPS.c -- i think we can trash those

Also seems to be used in two files in Libraries/StateEstimationFilters -- i think that directory can be trashed

There is useful stuff there but we can bring it back if needed in the future.